fdmi/plugins/m0scheduler

```diff
@@ -17,128 +17,277 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
+
+# This is FDMI plugin sample app wrapper that serves as a scheduler
+# for the FDMI records. It gathers that cluster infp using Hare hctl
+# tool, forks m0sched program and waits for records delivered by it.
+# For now it is just a demo that prints FDMI records to console, but
+# later it can perform records de-dup and replication coordination by
+# means of set of the replication python programs that perform actual
+# data move.
+
 import subprocess
 import signal
-from socket import gethostname
 import shlex
 import sys
 import json
-import ast
 import codecs
 
-kv_records = [] #global kv cache
+cluster_info = dict()  # global cluster info
+kv_records = []        # global kv cache
+processes = []         # global list if m0sched processes
```

Comment:
s/global list if m0sched processes/global list of m0sched processes
?

Reply:
agreed